src/components/__tests__/SDFlatButton.test.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import SDFlatButton from '../SDFlatButton.react';
+
+describe('SDFlatButton', () => {
```

Comment:
This needs a test to check click events are handled correctly. 

- Should at least check that a simulated click on the button causes `n_clicks` to increment correctly.
- We also should check that styling props work for the component, and that the children of the `SDFlatButton` work OK.

Reply:
I'll add those tests.